app/controllers/profiles_controller.rb

```diff
@@ -1,26 +1,26 @@
 # frozen_string_literal: true
 
 class ProfilesController < Devise::RegistrationsController
+  include ApiResponders
+
   prepend_before_action :authenticate_scope!, only: :update
   before_action :load_resource
 
   def update
     if resource.update_with_password(update_params)
       bypass_sign_in resource, scope: :user
-      render status: :ok, json: { notice: "User profile has been successfully updated", user: resource }
+      respond_with_success(t("successfully_updated", entity: "User profile"), :ok, { user: resource })
```

Comment:
No need to pass in `ok`. That should be the default value.

Reply:
@yedhink That would work if `status` was the last argument. If we don't pass `:ok`, then the user json that we pass will be considered to be the status which will lead to an error.